src/python/devcontainer-feature.json

```diff
@@ -67,7 +67,7 @@
     "PYTHON_PATH": "/usr/local/python/current",
     "PIPX_HOME": "/usr/local/py-utils",
     "PIPX_BIN_DIR": "/usr/local/py-utils/bin",
-    "PATH": "/usr/local/python/current/bin:/usr/local/py-utils/bin:${PATH}"
+    "PATH": "/usr/local/python/current/bin:/usr/local/py-utils/bin:/usr/local/share/nvm/current/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/vscode/.local/bin:${PATH}"
```

Comment:
The user name `vscode` is hard coded. Is that ideal?

Reply:
+1, This solution would prevent the feature from working for people who have specified the `containerUser`